package.json

```diff
@@ -2,84 +2,78 @@
-    "test:html": "node scripts/vnu-jar.js",
-    "test:unit": "tape \"tests/**/*.test.js\""
-  },
-  "repository": "nodejs/nodejs.org",
-  "keywords": [],
+  "repository": "https://github.com/nodejs/nodejs.org",
   "author": "Node.js Website Working Group",
   "license": "MIT",
   "engines": {
-    "node": ">=12"
+    "node": "v18"
+  },
+  "scripts": {
+    "serve": "next dev",
+    "build": "next build",
+    "postbuild": "next-sitemap",
+    "start": "next start",
+    "export": "next export -o build",
+    "deploy": "npm-run-all build export",
+    "lint:js": "eslint \"**/*.{js,jsx,ts,tsx}\" --cache --cache-file .eslintjscache --report-unused-disable-directives",
+    "lint:md": "eslint \"**/*.{md,mdx}\" --cache --cache-file .eslintmdcache",
+    "lint:scss": "stylelint --config .stylelintrc \"styles/**/*.{css,sass,scss}\"",
+    "lint": "npm-run-all lint:js lint:md lint:scss --parallel",
+    "lint:fix": "npm-run-all \"lint:js -- --fix\" \"lint:md -- --fix\" \"lint:scss -- --fix\" --parallel",
+    "prettier": "prettier \"**/*.{ts,tsx,js,scss,yml}\" --check",
+    "prettier:fix": "npm run prettier -- --write",
+    "format": "npm-run-all prettier:fix lint:fix",
+    "scripts:release-post": "node scripts/release-post/index.mjs"
   },
   "dependencies": {
-    "@metalsmith/markdown": "^1.6.0",
-    "@metalsmith/permalinks": "^2.4.1",
-    "autoprefixer": "^10.4.13",
-    "fs-extra": "^11.1.0",
-    "graceful-fs": "^4.2.10",
+    "@mdx-js/react": "^2.2.1",
+    "@next/font": "^13.1.1",
+    "classnames": "^2.3.2",
+    "feed": "^4.2.2",
+    "gray-matter": "^4.0.3",
     "handlebars": "^4.7.7",
-    "jquery": "^3.6.3",
-    "jquery.fancytable": "^1.0.33",
-    "jstransformer-handlebars": "^1.2.0",
-    "junk": "^3.1.0",
-    "lodash.defaultsdeep": "^4.6.1",
-    "marked": "^4.2.5",
-    "metalsmith": "^2.5.1",
-    "metalsmith-collections": "^0.9.0",
-    "metalsmith-feed": "^1.0.0",
-    "metalsmith-layouts": "^2.3.1",
-    "metalsmith-one-replace": "^0.1.7",
-    "metalsmith-yearly-pagination": "^4.0.2",
-    "ncp": "^2.0.0",
+    "highlight.js": "^11.7.0",
+    "isomorphic-dompurify": "^0.26.0",
+    "next": "^13.1.1",
+    "next-sitemap": "^3.1.45",
+    "nextra": "^2.2.13",
     "node-version-data": "^1.1.0",
     "npm-run-all": "^4.1.5",
-    "postcss": "^8.4.21",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-intl": "^6.2.5",
     "sass": "^1.57.1",
     "semver": "^7.3.8",
-    "strftime": "^0.10.1"
+    "strftime": "^0.10.1",
+    "swr": "^2.0.1"
   },
   "devDependencies": {
-    "chokidar": "^3.5.3",
+    "@builder.io/partytown": "^0.7.4",
+    "@next/eslint-plugin-next": "^13.1.1",
+    "@types/node": "^18.11.17",
+    "@types/react": "^18.0.26",
+    "@types/react-dom": "^18.0.10",
+    "@types/semver": "^7.3.13",
+    "@types/strftime": "^0.9.4",
+    "critters": "^0.0.16",
     "cross-env": "^7.0.3",
```

Comment:
this dependency is dead - no longer used in any scripts

Reply:
True, we could remove `cross-env`. Critter is also something we could remove for now, but will be used in the future for Service Workers (SW) with Google's Workbox, to enable offline-mode, PWAs, and other things.